pkg/cli/private_key_import_cfg.go

```diff
@@ -89,18 +94,51 @@ func (c *PrivateKeyImportCommand) Run(ctx context.Context, args []string) error
 	if err != nil {
 		return fmt.Errorf("encountered error when creating/getting key ring: %w", err)
 	}
-	logger.InfoContext(ctx, "Got key ring successfully\n", "key_ring", gotKeyRing.GetName())
+	logger.DebugContext(ctx, "Got key ring successfully\n", "key_ring", gotKeyRing.GetName())
 	gotKey, err := keyServer.GetOrCreateKey(ctx, c.cfg.ProjectID, c.cfg.Location, c.cfg.KeyRing, c.cfg.Key)
 	if err != nil {
 		return fmt.Errorf("encountered error when creating/getting key: %w", err)
 	}
-	logger.InfoContext(ctx, "Got key successfully\n", "key_ring", gotKey.GetName())
+	logger.DebugContext(ctx, "Got key successfully", "key_ring", gotKey.GetName())
 	gotImportJob, err := keyServer.GetOrCreateImportJob(ctx, c.cfg.ProjectID, c.cfg.Location, c.cfg.KeyRing, c.cfg.ImportJobPrefix)
 	if err != nil {
 		return fmt.Errorf("encountered error when creating/getting import job: %w", err)
 	}
-	logger.InfoContext(ctx, "Got import job successfully\n", "import_job", gotImportJob.GetName())
-	// TODO import key version
+	logger.DebugContext(ctx, "Got import job successfully", "import_job", gotImportJob.GetName())
+
+	if err := retry.Do(ctx, backoff, func(ctx context.Context) error {
```

Comment:
Unfortunately you can't re-use the backoff like this. The backoff is stateful, so the retry attempts from the first will get counted in the second attempt, which isn't what you want. You need to create a new backoff for each invocation of `retry.Do`. You can make it helper to inline it:

```go
func newBackoff() retry.Backoff {
  return retry.WithMaxRetries(5, retry.NewFibonacci(500*time.Millisecond))
}

// ...

retry.Do(ctx, newBackoff(), func(ctx context.Context) error {
  // ...
})
```

Reply:
Thanks for catching it! Didn't know `go-retry` was stateful.